scripts/nginx/preconfigure_nginx.py

```diff
@@ -17,23 +17,21 @@
 
 def _set_nginx_config_file(host_ip):
     config_file = os.path.join(os.path.expanduser('~'), 'djangosample/src/mysite/nginx.conf')
+    project_root_dir = os.path.join(os.path.expanduser('~'), 'djangosample/src')
```

Comment:
although i would generally first calculate project_root_dir
and then have config_file relative to it using os.path.join